lib/tasks/populate.rake

```diff
@@ -96,9 +121,9 @@ namespace :pvb do
 
     desc 'populate visits for review apps or dev environments'
     task visits: :environment do
-      %w[Sudbury Pentonville Usk].each do |prison_name|
+      %w[Sudbury Pentonville Usk Aylesbury].each do |prison_name|
         prison = Prison.find_by!(name: prison_name)
-        visits = 0.upto(14).map { |_i|
+        visits = 0.upto(140).map { |_i|
```

Comment:
The reason it got my attention is that review apps have been flakey in Heroku, we seem to be hitting rate limiting on one of the addons, I think it was Mailgun.

If review apps can be deployed consistently then I also prefer to have a larger dataset in review apps..

I should have mentioned this is my previous message